.vscode/cspell.json

```diff
@@ -933,6 +932,18 @@
         "responsed"
       ]
     },
+    {
+      "filename": "sdk/cognitiveservices/azure-cognitiveservices-vision-computervision/CHANGELOG.md",
+      "words": [
+        "prefered"
```

Comment:
I think we can make this spelling correction on the Changelog. Would you mind updating it?